### PR TITLE
Lanbao CM8JL65 is discontinued

### PR DESCRIPTION
**Pull request by hamishwillee**
Add markup to show that. Doing here in case #3448 does not merge.

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -247,7 +247,7 @@
       - [Benewake TFmini Lidar](sensor/tfmini.md)
       - [Lidar-Lite](sensor/lidar_lite.md)
       - [TeraRanger](sensor/teraranger.md)
-      - [Lanbao PSK-CM8JL65-CC5](sensor/cm8jl65_ir_distance_sensor.md)
+      - [✘ Lanbao PSK-CM8JL65-CC5](sensor/cm8jl65_ir_distance_sensor.md)
```
Comment (hamishwillee) [inline en/SUMMARY.md:250]: Note, trialing this markup for discontinued/don't use sidebar hinting.

File: en/sensor/cm8jl65_ir_distance_sensor.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
-# Lanbao PSK-CM8JL65-CC5 ToF Infrared Distance Measuring Sensor
+# Lanbao PSK-CM8JL65-CC5 ToF Infrared Distance Measuring Sensor (Discontinued)
+
+<Badge type="info" text="Discontinued" />
+
+:::warning
+This product has been [discontinued](../flight_controller/autopilot_experimental.md) and is no longer commercially available.
```
Comment (Claudio-Chies) [inline en/sensor/cm8jl65_ir_distance_sensor.md:6]: @hamishwillee 
You link here to the "Autopilot Experimental" Page, is this intended?
As there is no info in regard of discontinued products on that page.

Comment (hamishwillee) [reply en/sensor/cm8jl65_ir_distance_sensor.md:6]: No, that's a copy-paste error. Will fix now. Good catch.

Comment (hamishwillee) [reply en/sensor/cm8jl65_ir_distance_sensor.md:6]: Fixed in https://github.com/PX4/PX4-user_guide/pull/3498

**Pull request merged**